### DISCUSSION TITLE
Version 1.25.1

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#402] [Announcements]
- Bug fixes

### Important note:
My F-Droid repository has moved. If you are a user of my F-Droid repository (https://github.com/MM2-0/fdroid / https://raw.githubusercontent.com/MM2-0/fdroid/main/fdroid/repo/), please make sure to update the URL to https://fdroid.mm20.de/repo. The old repo will soon be discontinued.

(No action is required if you retrieve the app through Github, or the IzzyOnDroid or F-Droid main repository.)

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.25.1'>Version 1.25.1</a>.</em>

**Comment by NicholasUKRI** [comment]
Hey. Love the launcher.

I am having difficulties hiding icons from the App Page that are in my favourites though.

I.e. If I hide it from the App Page (as the icon is already on my front page in the favourites widget), it removes it from my favourites. Is there any way to keep an app in my favourites but hide it in the app page?
